scripts/twister

```diff
@@ -903,6 +908,11 @@ def main():
     if options.show_footprint or options.compare_report or options.release:
         options.enable_size_report = True
 
+    if options.seed == 0:
+        options.seed = random.randrange(8192)
```

Comment:
I'm by no means an expert in Python, but 0 seems like a valid seed, can we instead (above) set `default=random.randrange(8192)`?

Reply:
yeah, 0 can be a valid seed. Let me fix this.